nltk/tokenize/stanford_segmenter.py

```diff
@@ -75,8 +77,6 @@ def __init__(self,
 
         # This is passed to java as the -cp option, the old version of segmenter needs slf4j.
         # The new version of stanford-segmenter-2016-10-31 doesn't need slf4j
-        # When using old version, set path_to_slf4j='slf4j-api.jar' in the initilization of StanfordSegmenter
-        # The initilization program will search for your slf4j-api.jar according to the environment parameters you set
         self._stanford_jar = os.pathsep.join(
             [_ for _ in [stanford_segmenter, slf4j] if not _ is None])
```

Comment:
And I am sure that a relative path would not work. When you execute find_jar(), whose second parameter is path_to_slf4j, it would directly search according to the path without environment parameters and then terminate. So we have 3 choice:
A. let user input full path
B. set the second parameter of find_jar() to None when path_to_slf4j is not None, which means that path user input is just a signal to make sure that it's old version
C. revise the find_jar_iter() function in internals.py